releases/net471/KnownIssues/493108-WPF InvalidCastException when Resolving DynamicResource.md

```diff
@@ -0,0 +1,16 @@
+## WPF InvalidCastException when Resolving DynamicResource
+
+### Symptoms
+The crash occurs when a DynamicResource reference to a value A, in a context where a different value B is expected.� The values have different types, leading to an InvalidCastException.
```

Comment:
InvalidCastException here and below should be a link: `T:System.InvalidCastException`